### PR TITLE
Create Contextual Greeting Panel

### PR DESCRIPTION
**Pull request by Raymondsquared**
### Background
As part of dashboard redesign project, we wanted to start with the contextual greeting panel initially to make sure we are able to test the flow

### Detail
Create Contextual Greeting Panel 

### JIRA 
https://jira.qantas.com.au/browse/NAL-12

### UI

Old (Placeholder):
Screen Shot 2021-11-30 at 1 17 20 am

New (Component):
Screen Shot 2021-11-30 at 1 17 03 am

### CODE REVIEW

File: components/ContextualGreetingPanel/styles.ts [hunk 1]
```diff
@@ -13,3 +13,8 @@ export const GreetingPlaceHolder = styled.div`
     margin: 0;
   }
 `
+
+export const H1 = styled.h1`
+  font-family: Ciutadella Regular, Helvetica, Arial Narrow, Arial, sans-serif;
+  color: ${p => p.theme.color.primary};
+`
```
Comment (Raymondsquared) [inline components/ContextualGreetingPanel/styles.ts:20]: Not sure if we need the H1 just for this specific component

File: pages/index.tsx [hunk 2]
```diff
@@ -12,10 +13,7 @@ const Home: NextPage = () => {
         <meta name="description" content="Dashboard" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <GreetingPlaceHolder>
-        <H1>Good morning, Helena </H1>
-        <p>Great Business Pty Ltd . ABN 12 345 678 901</p>
-      </GreetingPlaceHolder>
+      <ContextualGreetingPanel abn={"12 345 678 901"} companyName={"Great Business Pty Ltd "} name={"Helena"} />
```
Comment (Raymondsquared) [inline pages/index.tsx:16]: I am thinking of passing an object here (as a prop) instead of individuals

File: pages/index.tsx [hunk 3]
```diff
@@ -12,10 +13,7 @@ const Home: NextPage = () => {
         <meta name="description" content="Dashboard" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <GreetingPlaceHolder>
-        <H1>Good morning, Helena </H1>
-        <p>Great Business Pty Ltd . ABN 12 345 678 901</p>
-      </GreetingPlaceHolder>
+      <ContextualGreetingPanel abn={"12 345 678 901"} companyName={"Great Business Pty Ltd "} userFirstName={"Helena"} />
```
Comment (Raymondsquared) [inline pages/index.tsx:16]: I am thinking of passing an object here (as a prop) instead of individuals

Comment (nicklaycock) [reply pages/index.tsx:16]: I think that  a prop is the best solution.

File: components/ContextualGreetingPanel/styles.ts (hunk 1 above)
Comment (ofhope) [reply components/ContextualGreetingPanel/styles.ts:20]: I think we can import it with `import { H1 } from '../../styles'`

File: pages/index.tsx (hunk 3 above)
Comment (ofhope) [reply pages/index.tsx:16]: Either or 🤷🏽‍♂️ 
Individual props are explicit and show up in  storybook controls easily.

**Review by ofhope — APPROVED** [review]

**Pull request merged**